### PR TITLE
BZ1996790:added not included to Gatekeeper

### PR DESCRIPTION
**Pull request by Amrita42**
- Applies to 4.6+ versions
- [Bugzilla](https://bugzilla.redhat.com/show_bug.cgi?id=1996790) 
- [Preview](http://file.del.redhat.com/asakthiv/BZ1996790/welcome/oke_about.html) >> table 2

### CODE REVIEW

**Comment by robszumski** [comment]
Looks good to me!

**Comment by Amrita42** [comment]
@showeimer please ack adding Klusterlet

**Review by abhatt-rh — COMMENTED** [review]
Hi Amrita, I just have some nits; else LGTM

File: welcome/oke_about.adoc [hunk 1]
```diff
@@ -241,8 +241,8 @@ The following table is a summary of the feature availability in {oke} and {produ
 | OpenShift Virtualization | Included | Included | OpenShift Virtualization Operator
 | Compliance Operator provided by Red Hat | Included | Included | Compliance Operator
 | File Integrity Operator | Included | Included | File Integrity Operator
-| Gatekeeper Operator | Included | Included | Gatekeeper Operator
-| Klusterlet provided by Red Hat | Included | Included | N/A
+| Gatekeeper Operator | Not Included - Requires separate subscription | Not Included - Requires separate subscription | Gatekeeper Operator
+| Klusterlet | Not Included - Requires separate subscription | Not Included - Requires separate subscription | N/A
```
[the remark this answers is not in the document]
Comment (Amrita42) [reply welcome/oke_about.adoc:245]: As discussed we are retaining the original content.

**Comment by showeimer** [comment]
I don't have permission to add a label, but LGTM!  Ty!

**Comment by Amrita42** [comment]
> I don't have permission to add a label, but LGTM! Ty!

No worries this will suiffice , thanks .

**Comment by abhatt-rh** [comment]
LGTM

**Comment by ahardin-rh** [comment]
LGTM!

**Pull request merged**